templates/Dockerfile.template

```diff
@@ -7,23 +7,35 @@ FROM {{ .From }}
 # add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
 RUN groupadd -r redis && useradd -r -g redis redis
 
+RUN set -ex; \
```

Comment:
Please change indentation from spaces to tabs to follow convention from the rest of the file.

Reply:
Done